### ISSUE TITLE
proxy-only-subnet for GKE Ingress

### ISSUE THREAD

**Issue by dbaid** [GoogleCloudPlatform/gke-networking-recipes#188]
I think you lack the creation of proxy-only-subnet for your case of GKE Ingress

**Comment by boredabdel** [comment]
Hello,

Which recipe is this referring to ?

**Comment by sawsa307** [comment]
I think it is missing in the ingress ILB recipe. We can include https://cloud.google.com/load-balancing/docs/l7-internal/setting-up-l7-internal#configure-a-network in the prerequisite for this recipe.

**Comment by sawsa307** [comment]
That is my guess. @dbaid can you confirm?

**Comment by dbaid** [comment]
> That is my guess. @dbaid can you confirm?

Yes. This recipe  https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/ingress/single-cluster/ingress-internal-basic. The topic of GKE Ingress internal loadbalance gets a lot of reference documents. I tested a private cluster through my major reference document -- https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balance-ingress . I always feel confused when studying gcp techniques. Too many things need to be studied. Step by step tests. 1. auto pilot cluster according the doc (in VPC custom subnet) 2. private cluster + cloud nat 3. cluster with least privilege service account 4.  internal lb with static ip 5. My final goal, GKE with Internal ingress in a shared VPC (still under test because no testing environment).

Regards,
Steven
